Jenkinsfile_CNP

```diff
@@ -8,13 +8,17 @@ def component = "task-configuration-api"
 
 def secrets = [
   'wa-${env}' : [
-      secret('wa-test-law-firm-a-username', 'TEST_WA_LAW_FIRM_USERNAME'),
-      secret('wa-test-law-firm-a-password', 'TEST_WA_LAW_FIRM_PASSWORD'),
-      secret('wa-idam-client-id', 'WA_IDAM_CLIENT_ID'),
-      secret('wa-idam-client-secret', 'WA_IDAM_CLIENT_SECRET'),
+    secret('wa-test-law-firm-a-username', 'TEST_WA_LAW_FIRM_USERNAME'),
+    secret('wa-test-law-firm-a-password', 'TEST_WA_LAW_FIRM_PASSWORD'),
+
```

Comment:
we don't need this one anymore

Reply:
We do, this is used in the FT to create CCD cases. Ravi and Marcin agreed to do it this way.